source/fundamentals/crud/write-operations/embedded-arrays.txt

```diff
@@ -173,92 +189,125 @@ that matches your query, use the filtered positional operator
+
+   .. code-block:: javascript
+      
+      arrayFilters: [
+        { "<identifier>.<arrayField1>": <updateParameter1> },
+        { "<identifier>.<arrayField2>": <updateParameter2> },
+        ...
+      ]
+
+#. Pass the query document, the update document, and options to an
+   update method. The following sample code shows how to call the
+   ``updateOne()`` method with these parameters:
+
+   .. code-block:: javascript
+   
+      await myColl.updateOne(query, updateDocument, options);
 
 Example
 ```````
 
-To add a "garlic" topping to certain order items, format your update
-document as follows:
-
-.. code-block:: javascript
-
-   { $push: { items.$[orderItem].toppings: "garlic" } }
-
-This update document specifies the following:
-
-- ``$push``: the update operator
-- ``items``: the array in the document to update
-- ``orderItem``: the identifier for the filtered positional operator
-- ``toppings``: the field on the ``items`` array element to update
-- ``garlic``: the value to push onto the ``toppings`` array
-
-Next, add the matching criteria in your ``arrayFilters`` object in your update
-operation's ``options`` parameter. This object is an array of queries
-that specify which array elements to include in the update. To add the
-"garlic" topping to order items of type "pizza" and "large size", pass the
-following ``arrayFilters``:
-
-.. code-block:: javascript
-
-   arrayFilters: [
-     { orderItem.type: "pizza" },
-     { orderItem.size: "large" }
-   ]
-
-The following snippet shows the complete update method:
-
-.. literalinclude:: /code-snippets/crud/arrayFilters.js
-   :language: javascript
-   :start-after: start arrayFiltersOne example
-   :end-before: end arrayFiltersOne example
-
-After you run the method, your customer document for Steve
-Lobsters resembles the following:
+This example shows how to update certain matching array elements in the
+following sample documents, which describe shopping lists made for specific
+recipes:
 
 .. code-block:: javascript
    :copyable: false
 
    {
-     name: "Steve Lobsters",
-     address: "731 Yexington Avenue",
+     _id: ...,
+     date: "11/12/2023",
+     items: [
+       { item: "Scallions", quantity: 3, recipe: "Fried rice" },
+       { item: "Mangos", quantity: 4, recipe: "Salsa" },
+       { item: "Pork shoulder", quantity: 1, recipe: "Fried rice" },
+       { item: "Sesame oil", quantity: 1, recipe: "Fried rice" }
+     ]
+   },
+   {
+     _id: ...,
+     date: "11/20/2023",
      items: [
-       { type: "pizza", size: "large", toppings: ["pepperoni", "garlic"] },
-       { type: "pizza", size: "large", toppings: ["pineapple", "ham", "garlic"], ...},
-       ...
+       { item: "Coffee beans", quantity: 1, recipe: "Coffee" }
      ]
    }
 
-Example
-```````
+Suppose you want to increase the quantity of items you purchase for a
+recipe on your ``"11/12/2023"`` grocery trip. You want to double the quantity if
+the item meets the following criteria:
```

Comment:
Suggestion:

I think for clarity that it has to match all the criteria (conjunction) rather than just one (disjunction) would be helpful:

```suggestion
the item meets all the following criteria:
```